coffee_github/src/repository.rs

```diff
@@ -10,7 +10,7 @@ use walkdir::DirEntry;
 use walkdir::WalkDir;
 
 use coffee_lib::errors::CoffeeError;
-use coffee_lib::macros::error;
+use coffee_lib::macros::{commit_id, error};
```

Comment:
in rust there is a not written concept of prelude, where you can do `coffee_lib::prelude::*` and this include all the default configuration of it

I do not know if this will be the case, but I think if this macros start to grow we can think to make something like this

Reply:
I think for now at least we don't need this?
if we do `use coffee_lib::macros::*;` we would have to change every occurrence of `error!` to `macros::error!`

Reply:
yeah just something to keep in mind